### PR TITLE
spacing in let-in

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

File: test_data/curried_fn.good.nix [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
   mkValueStringDefault = {}: v: with builtins;
-    let err = t: v: abort
-      "oups";
-    in 92;
```
Comment (matklad) [inline test_data/curried_fn.good.nix]: @zimbatm not sure what rule do we want here. The current implementation enforces line breaks if let itself is multiline, but we can allow `in 92` to be on the single line.

Comment (zimbatm) [reply test_data/curried_fn.good.nix]: I think it's fine like that

**Comment by matklad** [comment]
hm, why this is still in queu?

**Pull request merged**